conan/api/model.py

```diff
@@ -81,15 +81,22 @@ def prefs(self):
         return prefs
 
     def add_prefs(self, prefs, configurations=None):
-        confs = configurations or {}
+        confs = configurations or OrderedDict()
         for pref in prefs:
-            binary_info = confs.get(pref, {})
+            binary_info = confs.get(pref, OrderedDict())
             self.recipes.setdefault(pref.ref, []).append((pref, binary_info))
 
+    @property
+    def ordered_recipes_by_name(self):
+        ret = OrderedDict()
```

Comment:
This is not a ``SortedDict``, the ``OrderedDict`` is ordered by insertion order, so I think this is not really sorting anything?